sdc/sdc_autogenerated.py

```diff
@@ -1516,3 +1510,267 @@ def _series_operator_eq_common_impl(self, other):
             return _series_operator_eq_common_impl
 
     return None
+
+
+@sdc_overload(operator.lt)
+def sdc_str_arr_operator_lt(self, other):
+
+    self_is_str_arr = self == string_array_type
+    other_is_str_arr = other == string_array_type
+    operands_are_arrays = self_is_str_arr and other_is_str_arr
+
+    if not (operands_are_arrays
+            or (self_is_str_arr and isinstance(other, types.UnicodeType))
+            or (isinstance(self, types.UnicodeType) and other_is_str_arr)):
+        return None
+
+    if operands_are_arrays:
+        def _sdc_str_arr_operator_lt_impl(self, other):
+            if len(self) != len(other):
+                raise ValueError("Mismatch of String Arrays sizes in operator.lt")
+            n = len(self)
+            out_list = [False] * n
+            for i in numba.prange(n):
+                out_list[i] = out_list[i] = (self[i] < other[i]
```

Comment:
Please apply such fix in all the places.
```suggestion
                out_list[i] = (self[i] < other[i]
```

Reply:
Thank you, fixed it.